### DISCUSSION TITLE
GridSplitter support in WinUI 3 Desktop - Windows App SDK 1.0.0

### DISCUSSION THREAD

**Discussion by JosHuybrighs** [microsoft/WindowsAppSDK#2254] [Q&A] [answered]
I need `GridSplitter `for an app that I am porting from UWP/DesktopBridge to WinUI-Desktop (MSIX packaged).
I started with getting the `Microsoft.Toolkit.Uwp.UI.Control` nuget package, but there doesn't seem to be any `GridSplitter `in there.
The package mentions UWP in its name what could suggest that it doesn't work with a WinUI-Desktop app.

So the question: does WinUI-Desktop require its own `GridSplitter` and if so is it already available?
I am targetting .NET 6 and use C#.

**Comment by kmgallahan** [comment, marked as answer]
https://portal.productboard.com/winappsdk/1-windows-app-sdk/c/39-splitter-control

https://docs.microsoft.com/en-us/dotnet/api/communitytoolkit.winui.ui.controls.gridsplitter?view=win-comm-toolkit-dotnet-7.0
